src/library/scala/collection/mutable/Iterable.scala

```diff
@@ -17,17 +17,6 @@ trait Iterable[A]
 trait IterableOps[A, +CC[X], +C]
   extends collection.IterableOps[A, CC, C] {
 
-  /** Modifies this $coll by applying a function to all elements of this $coll.
-    *
-    *  @param f      the function to apply to each element.
-    *  @return       this $coll modified by replacing all elements with the
-    *                result of applying the given function `f` to each element
-    *                of this $coll.
-    */
-  def mapInPlace(f: A => A): this.type
-
-  @deprecated("Use `mapInPlace` instead", "2.13.0")
-  @`inline`final def transform(f: A => A): this.type = mapInPlace(f)
```

Comment:
@lrytz Why did we have this `transform` operation on `mutable.Iterable`? It does not seem to exist in 2.12.

Reply:
https://github.com/scala/scala/blob/v2.12.6/src/library/scala/collection/mutable/SeqLike.scala#L43 -- maybe the method should move to `mutable.SeqOps`? It's used in some tests (t4813.scala). The deprecated version here was added in https://github.com/scala/collection-strawman/pull/452.

Reply:
I agree with moving to `mutable.SeqOps`. (And that’s what’s done in this PR)